conda_mirror/conda_mirror.py

```diff
@@ -678,6 +679,10 @@ def main(upstream_channel, target_directory, temp_directory, platform,
     with tempfile.TemporaryDirectory(dir=temp_directory) as download_dir:
         logger.info('downloading to the tempdir %s', download_dir)
         for package_name in sorted(to_mirror):
+            disk = os.statvfs(download_dir)
```

Comment:
https://docs.python.org/3/library/os.html#os.fstatvfs suggests this function is Linux-only. Are you aware of a cross-platform solution for Windows users (which I think we have)?

Reply:
We only test for 3.4+ and up (https://github.com/Valassis-Digital-Media/conda-mirror/blob/master/.travis.yml) and declare it's a Python 3 only package (https://github.com/Valassis-Digital-Media/conda-mirror#compatibility) so I think shutil.disk_usage sounds perfect.